Makefile

```diff
@@ -1,113 +1,99 @@
-OPENOCD_BIN?=$(CONDA_DIR)/bin/openocd
-DIGILENT_CONFIG=openocd-digilent-basys3.cfg
+DIGILENT_CONFIG=openocd-digilent-arty7.cfg
 
+PROJECT=blink
 
 all: top.bit
 
 %.eblif : %.v synth.tcl
-	symbiflow_arch_defs_SOURCE_DIR=$(SYMBIFLOW_ARCH_DEFS) \
-        OUT_EBLIF=$@ \
-        $(YOSYS_BIN) -q -p "tcl synth.tcl" -l $@.log $<
+	symbiflow_arch_defs_SOURCE_DIR=$(SYMBIFLOW_ARCH_DEFS) OUT_EBLIF=$@ \
+		$(YOSYS_BIN) -q -p "tcl synth.tcl" -l $@.log $<
 
-%_io.place : %.eblif
-	$(IOPLACE) --pcf $(PCF) --map $(MAP) --blif $^ --out $@
+%_io.place : %.eblif %.net
+	$(IOPLACE) --pcf $(PCF) --map $(ARCH_PINMAP) --blif $< --net $*.net  --out $@
 
 %.net: %.eblif
-	$(VPR_BIN) $(ARCH_TIMING_XML) $< --device $(DEVICE) --read_rr_graph $(RR_GRAPH) $(VPR_PARAMS) --pack > $*-pack.log
+	$(VPR_BIN) $(ARCH_TIMING) $< --device $(DEVICE) --read_rr_graph $(RR_GRAPH) $(VPR_PARAMS) --pack > $*-pack.log
 
 %.place: %.eblif %_io.place %.net
-	$(VPR_BIN) $(ARCH_TIMING_XML) $*.eblif --device $(DEVICE) --read_rr_graph $(RR_GRAPH) $(VPR_PARAMS) --fix_pins $*_io.place --place > $*-place.log
+	$(VPR_BIN) $(ARCH_TIMING) $*.eblif\
+		--device $(DEVICE) \
+		--read_rr_graph $(RR_GRAPH) \
+		--fix_pins $*_io.place --place \
+		$(VPR_PARAMS) > $*-place.log
 
 %.route: %.eblif %.place
-	$(VPR_BIN) $(ARCH_TIMING_XML) $*.eblif --device $(DEVICE) --read_rr_graph $(RR_GRAPH) $(VPR_PARAMS) --route > $*-route.log
+	$(VPR_BIN) $(ARCH_TIMING) $*.eblif \
+		--device $(DEVICE) \
+		--read_rr_graph $(RR_GRAPH) \
+		$(VPR_PARAMS) --route > $*-route.log
 
 top.fasm: $(PROJECT).eblif $(PROJECT).place $(PROJECT).route $(PROJECT).net
-	$(GENFASM_BIN) $(ARCH_TIMING_XML) $(PROJECT).eblif --device $(DEVICE) --read_rr_graph $(RR_GRAPH) $(VPR_PARAMS) > fasm.log
+	$(GENFASM_BIN) $(ARCH_TIMING) $(PROJECT).eblif \
+		--device $(DEVICE) \
+		$(VPR_PARAMS) \
+		--read_rr_graph $(RR_GRAPH)> fasm.log
 
 top.frames: top.fasm
-	$(FASM2FRAMES) --db-root $(SYMBIFLOW_ARCH_DEFS)/third_party/prjxray-db/artix7 --sparse --roi $(SYMBIFLOW_ARCH_DEFS)/third_party/prjxray-db/artix7/harness/basys3/swbut/design.json $< $@
+	$(FASM2FRAMES) \
+		--db-root $(SYMBIFLOW_ARCH_DEFS)/third_party/prjxray-db/artix7 \
+		--sparse --roi $(SYMBIFLOW_ARCH_DEFS)/third_party/prjxray-db/artix7/harness/arty-a7/swbut/design.json $< $@
 
 %.bit: %.frames
-	$(XC7PATCH) --frm_file $^ --output_file $@ --bitstream_file $(SYMBIFLOW_ARCH_DEFS)/third_party/prjxray-db/artix7/harness/basys3/swbut/design.bit --part_name xc7a35tcpg236-1 --part_file $(SYMBIFLOW_ARCH_DEFS)/third_party/prjxray-db/artix7/xc7a35tcpg236-1.yaml
+	$(XC7PATCH) --frm_file $^ \
+		--output_file $@ \
+		--bitstream_file $(SYMBIFLOW_ARCH_DEFS)/third_party/prjxray-db/artix7/harness/arty-a7/swbut/design.bit \
+		--part_name xc7a35tcpg236-1 --part_file $(SYMBIFLOW_ARCH_DEFS)/third_party/prjxray-db/artix7/xc7a35tcpg236-1.yaml
 
 prog: top.bit
 	$(OPENOCD_BIN) -f $(DIGILENT_CONFIG) -c "init ; pld load 0 $^ ; exit"
 
 clean:
-	rm -f $(PROJECT).eblif $(PROJECT)_synth.v
```

Comment:
I'd keep this very specific 'make clean' rules, so that it is clear for someone trying this what actually got generated (this this is meant to be a Makefile to understand how things hang together)

Reply:
(solved)